anvill/src/Lifters/FunctionLifter.cpp

```diff
@@ -115,6 +116,31 @@ static void MuteStateEscape(llvm::CallInst *call) {
   call->setArgOperand(remill::kStatePointerArgNum, undef_val);
 }
 
+static llvm::Function *GetAnvillSwitchFunc(llvm::Module &module,
+                                           bool complete) {
+
+  auto func_name =
+      complete ? kAnvillSwitchCompleteFunc : kAnvillSwitchIncompleteFunc;
+
+  if (auto func = module.getFunction(func_name)) {
+    return func;
+  }
+
+  auto &context = module.getContext();
+  auto return_type = llvm::Type::getInt64Ty(context);
+  const std::vector<llvm::Type *> func_parameters = {
+      llvm::Type::getInt64Ty(context)};
+
+  auto func_type = llvm::FunctionType::get(return_type, func_parameters, true);
+
+  auto func = llvm::Function::Create(
+      func_type, llvm::GlobalValue::ExternalLinkage, func_name.c_str(), module);
```

Comment:
I don't think you need to do `.c_str()`. It should be an `llvm::StringRef`, which can be implicitly constructed from a `std::string`.